<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginMenu</class>
 <widget class="QDialog" name="LoginMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QWidget" name="Login">
    <widget class="QLineEdit" name="loginPass">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="loginLoginButton">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>310</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginRegisterButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>220</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Not Registered?</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="loginUser">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="loginErrorLabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Register">
    <widget class="QLabel" name="registerErrorLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registerUser">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="registerMainPass">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="loginLoginButton_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>290</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="registerCheckPass">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>171</width>
       <height>27</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="registerBackToLogin">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>380</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Go to Login</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>loginUser</tabstop>
  <tabstop>loginPass</tabstop>
  <tabstop>loginLoginButton</tabstop>
  <tabstop>registerUser</tabstop>
  <tabstop>registerMainPass</tabstop>
  <tabstop>registerCheckPass</tabstop>
  <tabstop>loginLoginButton_2</tabstop>
  <tabstop>registerBackToLogin</tabstop>
  <tabstop>loginRegisterButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
